pkgs/development/python-modules/flet/pyproject.toml.patch

```diff
@@ -1,11 +1,15 @@
 --- a/pyproject.toml
 +++ b/pyproject.toml
-@@ -20,7 +20,7 @@ flet-core = "0.6.2"
+@@ -20,10 +20,10 @@
  python = "^3.7"
  typing-extensions = { version = "^4.4.0", python = "<3.8" }
  websocket-client = "^1.4.2"
 -watchdog = "^2.2.1"
 +watchdog = ">=2.2.1"
  oauthlib = "^3.2.2"
  websockets = "^10.4"
- httpx = "^0.23.3"
+-httpx = "^0.23.3"
++httpx = ">=0.23.3"
+ packaging = "^23.0"
```

Comment:
Please move this into postPatch so that we don't need to update the patch every update

Reply:
Thanks for the feedback. Done.